### PR TITLE
Remove isset check when constructor will not throw

### PR DESCRIPTION
**Pull request by michaelbausor**

### CODE REVIEW

File: src/main/resources/com/google/api/codegen/php/method_sample.snip [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+@snippet decorateSampleCode(xapiClass, apiMethod, coreSampleCode)
     try {
-        ${@apiMethod.apiVariableName} = new {@apiMethod.apiClassName}();
+        @if xapiClass.hasMissingDefaultOptions
+            $options = [
+                @if xapiClass.missingDefaultServiceAddress
+                    'serviceAddress' => 'my-service-address',
+                @end
+                @if xapiClass.missingDefaultServiceScopes
+                    'scopes' => ['my-service-scope'],
```
Comment (garrettjonesgoogle) [inline src/main/resources/com/google/api/codegen/php/method_sample.snip]: Why not make my-service-scope and my-service-address defaults if you still have to set them here?

Comment (michaelbausor) [inline src/main/resources/com/google/api/codegen/php/method_sample.snip]: I'm not sure I know what you mean - my-service-address and my-service-scope are placeholders in the method sample, which the user has to set to the appropriate values. Maybe they should be in [MY-SERVICE-ADDRESS] format?

Comment (garrettjonesgoogle) [inline src/main/resources/com/google/api/codegen/php/method_sample.snip]: oh! I get it. Nevermind.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**